soccer/gameplay/root_play.py

```diff
@@ -80,19 +80,22 @@ def execute_running(self):
             if self.play != None:
                 if self.play.__class__ not in map(lambda tup: tup[0],
                                                   enabled_plays_and_scores):
-                    logging.info("Current play '" + self.play.__class__.
-                                 __name__ + "' no longer enabled, aborting")
+                    logging.info("Current play '" +
```

Comment:
While I prefer these changes, they might get overridden by checkstyle later, if it leaves them, thats great!